### ISSUE TITLE
Swift 3.0 Branch

### ISSUE THREAD

**Issue by twodayslate** [dankogai/swift-pons#6]
Swift 3.0's release is soon. XCode's auto update does not cut it.

**Comment by mgriebling** [comment]
Yes, same issue here.  Tried updating myself but got hopelessly lost in all the inheritance mess.

**Comment by primetimer** [comment]
Wow Swift3 Support would be really cool. Tried it by myself, but get lost....

**Comment by zackshapiro** [comment]
Is this project still alive?
